mllib/src/main/scala/org/apache/spark/mllib/tree/impurity/Entropy.scala

```diff
@@ -94,6 +94,9 @@ private[tree] class EntropyAggregator(numClasses: Int)
       throw new IllegalArgumentException(s"EntropyAggregator given label $label" +
         s" but requires label < numClasses (= $statsSize).")
     }
+    if (label < 0) {
+      throw new IllegalArgumentException(s"Entropy does not support negative labels")
```

Comment:
Shall we put `label` value to the error message? Add `.` to the end of the sentence.